modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinySelections.ts

```diff
@@ -4,6 +4,15 @@ import { Editor } from '../../alien/EditorTypes';
 import { createDomSelection } from '../../selection/SelectionTools';
 import { TinyDom } from '../TinyDom';
 
+const setRawSelection = (editor: Editor, startPath: number[], soffset: number, finishPath: number[], foffset: number): void => {
+  const rng = createDomSelection(TinyDom.body(editor), startPath, soffset, finishPath, foffset);
+  const sel = editor.getWin().getSelection();
```

Comment:
This could also use `editor.selection.getSel()` if you prefer 🤷

Reply:
I looked at that. The `EditorSelection` interface returned from `Selection.ts` does include `getSel`:
https://github.com/tinymce/tinymce/blob/595b2d78f086ae408c9293bd97f5f2d47747385d/modules/tinymce/src/core/main/ts/api/dom/Selection.ts#L81

But the `Selection` interface used in the types does not:
https://github.com/tinymce/tinymce/blob/6bd1e25080967978ea3263ae75fbc0fb029ddfa9/modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts#L3-L11

Although I now realise that's an mcagar file, I assumed we were using the TinyMCE types. I'll change mcagar.

Reply:
We can't use TinyMCE types here, as it needs to a) work with multiple TinyMCE versions and b) cannot have a circular dependency.